packages/serverless-nextjs-component/serverless.js

```diff
@@ -238,7 +238,7 @@ class NextjsComponent extends Component {
       : process.cwd();
 
     const buildConfig = {
-      enabled: inputs.build && inputs.build.enabled !== false,
+      enabled: inputs.build ? inputs.build !== false && inputs.build.enabled !== false : true,
```

Comment:
!== false is the same as saying === true.

Reply:
`=== true` wouldn't allow objects to pass through to the second case, which is from the original code. So I'm trying to keep to the logic outlined in the original and in the docs. Only exactly false disables builds.